<?xml version="1.0" encoding="utf-8"?>
<launch>

  <node pkg="competition1" type="depthtest" name="depthtest_node">
    <rosparam file="$(find competition1)/config/circle_detector.yaml" command="load" />
  </node>

</launch>
